packages/design-system/src/components/separator.stories.tsx

```diff
@@ -1,9 +1,9 @@
-import { Separator } from "./separator";
```

Comment:
are you sure you wanna do this to separator?

Reply:
ofc not, edited the wrong file